webpack.config.js

```diff
@@ -1,19 +1,45 @@
 const path = require('path');
 
-module.exports = {
-  entry: './src/Crisp.jsx',
-  output: {
-    path: path.resolve('dist'),
-    filename: 'crisp.js',
-    libraryTarget: 'commonjs2',
+module.exports = [
+  {
+    entry: './src/Crisp.jsx',
+    output: {
+      path: path.resolve(__dirname, './dist/web/'),
+      filename: 'crisp-web.js',
+      library: {
+        type: 'commonjs2',
+      },
+    },
+    module: {
+      rules: [
+        {
+          test: /\.jsx?$/,
+          exclude: /(node_modules)/,
+          use: 'babel-loader',
+        },
+      ],
+    },
   },
-  module: {
-    rules: [
-      {
-        test: /\.jsx?$/,
-        exclude: /(node_modules)/,
-        use: 'babel-loader',
+  {
+    entry: './src/Crisp.jsx',
+    experiments: {
+      outputModule: true,
+    },
+    output: {
+      path: path.resolve(__dirname, './dist/module/'),
+      filename: 'crisp-module.js',
+      library: {
+        type: 'module',
```

Comment:
Why `module`? and not one of the 17 built-in other ways to generate a javascript file with webpack?

I'm not a huge fan of maintaining different versions of the same lib. Most frameworks support 'es'  and `commonjs1/2` import style. vitejs seems to have an issue with the latter, not sure we should support multiple formats just for one framework.

Reply:
You're write. So I close this PR.